Example/A person jumping

Kartik-Sama opened the pull request:
Your branch is up to date with 'origin/master'.
This example uses canvas to show a jumping person.
Changes to be committed:
	modified:   index.html

Commit:
On branch master
Your branch is up to date with 'origin/master'.

Changes to be committed:
	modified:   index.html

satu0king commented on index.html:
```diff
@@ -7,26 +7,67 @@
 </html>
 <script>
-
     // Declare all global variables here
```
Add your name as author here.

Kartik-Sama replied:
I have added a comment. Hope this is what you meant.

Kartik-Sama replied:
In my recent commit I have added my name. Hope this is what you asked for.

satu0king replied:
Yes but more like 
`Author - Kathik Sama`
I will merge for now

satu0king reviewed (changes requested):
Add your name and we can merge.

Commit:
On branch master
Your branch is up to date with 'origin/master'.

Changes to be committed:
	modified:   index.html

Kartik-Sama commented on index.html:
```diff
@@ -7,26 +7,67 @@
 
 </html>
 <script>
-
```
//Work by Karthik Sama
